src/docs/README.md

```diff
@@ -0,0 +1,40 @@
+# MIxS Repository Documentation
+This folder contains static documents that describe this repository, its editing policies, and workflows and guidance for editing.
+
+For information on GSC see [The GSC website](https://www.gensc.org/)
+
+For information on MIxS and its use, see [its autogenerated documentation](https://w3id.org/mixs).
+
+The GSC uses LinkML tooling to maintain the MIxS schema. For guidance on how to use LinkML or contribute to the core LinkML code, please see [the LinkML documentation](https://linkml.io/linkml/).
+
+## Code of Conduct
+
+The MIxS team strives to create a welcoming environment for editors, users and other contributors.
+Please carefully read our [Code of Conduct](../../CODE_OF_CONDUCT.md) before making a contribution.
+
+## Folder contents
+
+* README.md: This file
+* [about.md](about.md): A description of MIxS. This file is auto-generated by LinkML. For more information on MIxS, see [its autogenerated documentation](https://w3id.org/mixs).
+* [edit_worklow.md](edit_worklow.md): Guidance for members of the GSC Technical Working Group on how to request and create a new checklist or package, request and create a new term, update an existing term, or report an issue with the MIxS code
+* [external_contribution.md](external_contribution.md): Guidance and how-to's for external contributors, including how to request changes or report bugs
+* [policy.md](policy.md): Policies for editing, versioning, and communication about MIxS releases
+
+## GitHub Best Practices
+### How to write a great issue
+
+Please review GitHub's overview article,
+["Tracking Your Work with Issues"][about-issues].
+
+**Labeling issues:** Issue labeld were updated 2023/12. Please do not create new labels on your own. Instead, create an issue to request a new label, which will be reviewed by the TWG or CIG. See the [labels page](https://github.com/GenomicsStandardsConsortium/mixs/labels) for descriptions of existing issues.
```

Comment:
typo: "labeld" should be "labels" 
typo: "for descriptions of existing *issues* " should read "for descriptions of existing labels"